docs/serving/samples/grpc-ping-go/README.md

```diff
@@ -2,6 +2,13 @@ A simple gRPC server written in Go that you can use for testing.
 
 This sample requires knative/serving 0.4 or later.
 
+## Before you begin
+
```

Comment:
There is no intro sentence in this example ? you added a small sentence in all the other ones.

Reply:
I have added a intro sentence here